Update README with Creative Cloud installation guide (#25)
plainly-videos/after-effects-plugin: README.md

## Installation

### Through Creative Cloud

> 1. Check the plugin at [Adobe Exchange - Creative Cloud](https://exchange.adobe.com/apps/cc/202811/plainly-videos).

Author: ```suggestion
1. Acquire the plugin at [Adobe Exchange - Creative Cloud](https://exchange.adobe.com/apps/cc/202811/plainly-videos).
```

> 1. Acquire our plugin at [Adobe Exchange - Creative Cloud](https://exchange.adobe.com/apps/cc/202811/plainly-videos).
> 2. Open **Creative Cloud** application.
> 3. Navigate to **Stock & Marketplace** > **Plugins** > **All plugins**
> 4. Search for `plainly videos`, click install and proceed with an installation process.

Reviewer: Aquire is wrong or? Although check is not better, any better ideas?

```suggestion
1. Check the plugin at [Adobe Exchange - Creative Cloud](https://exchange.adobe.com/apps/cc/202811/plainly-videos).
2. Open **Creative Cloud** application.
3. Navigate to **Stock & Marketplace** > **Plugins** > **All plugins**
4. Search for `Plainly Videos`, click install and proceed with the installation process.
```
